### PR TITLE
GH-107 Multification setup

### PR DESCRIPTION
**Pull request by CitralFlo**
Resolve: #107

This pull request includes several changes to integrate the Multification library into the EternalCombat plugin, refactor notification handling, and remove unused code. The most important changes include adding Multification dependencies, updating the notification system, and removing the `FightBossBarService` class.

### Multification Integration:
* [`buildSrc/src/main/kotlin/Versions.kt`](diffhunk://#diff-c1c2c95e10d73bad89764533c4e4cd48653a02484f07df8316b8fa9fc15da0c8L12-R12): Added `MULTIFICATION` constant.
* [`eternalcombat-plugin/build.gradle.kts`](diffhunk://#diff-e925be4f07d19fb16fdf9efadb11edfa507a9ae2e7ccff4db515b5485c92164fR48-R51): Added Multification dependencies and updated `shadowJar` configuration. [[1]](diffhunk://#diff-e925be4f07d19fb16fdf9efadb11edfa507a9ae2e7ccff4db515b5485c92164fR48-R51) [[2]](diffhunk://#diff-e925be4f07d19fb16fdf9efadb11edfa507a9ae2e7ccff4db515b5485c92164fL95-R100)

### Notification System Refactor:
* [`eternalcombat-plugin/src/main/java/com/eternalcode/combat/EternalCombatReloadCommand.java`](diffhunk://#diff-7d67fe3a0c505315f796542abfc2956e5d124f937c4e40a1a2b4ee73309fa0d6R5-R6): Replaced `Formatter` with Multification `Notice` for reload messages. [[1]](diffhunk://#diff-7d67fe3a0c505315f796542abfc2956e5d124f937c4e40a1a2b4ee73309fa0d6R5-R6) [[2]](diffhunk://#diff-7d67fe3a0c505315f796542abfc2956e5d124f937c4e40a1a2b4ee73309fa0d6L13-R22) [[3]](diffhunk://#diff-7d67fe3a0c505315f796542abfc2956e5d124f937c4e40a1a2b4ee73309fa0d6L36-R44)
* [`eternalcombat-plugin/src/main/java/com/eternalcode/combat/config/implementation/PluginConfig.java`](diffhunk://#diff-44c84ce06f3b168d498a42d242351891e90bf2c487a1ee370003faa308d06e01L7-R13): Updated notification messages to use Multification `Notice`. [[1]](diffhunk://#diff-44c84ce06f3b168d498a42d242351891e90bf2c487a1ee370003faa308d06e01L7-R13) [[2]](diffhunk://#diff-44c84ce06f3b168d498a42d242351891e90bf2c487a1ee370003faa308d06e01L171-R302)

### Code Cleanup:
* [`eternalcombat-plugin/src/main/java/com/eternalcode/combat/CombatPlugin.java`](diffhunk://#diff-c3dcc21d34fdf1b6f4e0a8cd06801c4538b104f4dc1c532907c08627190bdbb7L28): Removed `FightBossBarService` and updated related code. [[1]](diffhunk://#diff-c3dcc21d34fdf1b6f4e0a8cd06801c4538b104f4dc1c532907c08627190bdbb7L28) [[2]](diffhunk://#diff-c3dcc21d34fdf1b6f4e0a8cd06801c4538b104f4dc1c532907c08627190bdbb7L111-R115) [[3]](diffhunk://#diff-c3dcc21d34fdf1b6f4e0a8cd06801c4538b104f4dc1c532907c08627190bdbb7L134-R132) [[4]](diffhunk://#diff-c3dcc21d34fdf1b6f4e0a8cd06801c4538b104f4dc1c532907c08627190bdbb7L156-R154)
* [`eternalcombat-plugin/src/main/java/com/eternalcode/combat/fight/FightTagCommand.java`](diffhunk://#diff-ac1a7f3aa79745a67415080284e59f3838460aca37b955e988b27e32ae82c7e5L19): Refactored to use Multification `Notice` for combat status and tag messages. [[1]](diffhunk://#diff-ac1a7f3aa79745a67415080284e59f3838460aca37b955e988b27e32ae82c7e5L19) [[2]](diffhunk://#diff-ac1a7f3aa79745a67415080284e59f3838460aca37b955e988b27e32ae82c7e5L38-L45) [[3]](diffhunk://#diff-ac1a7f3aa79745a67415080284e59f3838460aca37b955e988b27e32ae82c7e5L54-R74)

### CODE REVIEW

**Comment by CitralFlo** [comment]
Will be tested in the future, waiting for bossbar merge in Multification

**Review by Rollczi — APPROVED** [review]

**Review by vLuckyyy — APPROVED** [review]

File: eternalcombat-plugin/src/main/java/com/eternalcode/combat/EternalCombatReloadCommand.java [hunk 1]
```diff
@@ -10,13 +12,14 @@
...
 import java.time.Duration;
 import org.bukkit.command.CommandSender;
-import panda.utilities.text.Formatter;
 
 @Command(name = "combatlog", aliases = "combat")
 @Permission("eternalcombat.reload")
 public class EternalCombatReloadCommand {
 
-    private static final String RELOAD_MESSAGE = "<b><gradient:#8a1212:#fc6b03>EternalCombat:</gradient></b> Reloaded EternalCombat in <color:#fce303>{TIME}ms!</color>";
+    private static final Notice RELOAD_MESSAGE = BukkitNotice.builder()
```
Comment (P1otrulla) [inline eternalcombat-plugin/src/main/java/com/eternalcode/combat/EternalCombatReloadCommand.java:20]: w tym przypadku może być proste Notice -> BukkitNotice różni się tym że posiada dodatkowo metody stricte pod dzwieki

File: eternalcombat-plugin/src/main/java/com/eternalcode/combat/fight/tagout/FightTagOutCommand.java [hunk 2]
```diff
@@ -9,11 +9,10 @@
 import dev.rollczi.litecommands.annotations.execute.Execute;
 import dev.rollczi.litecommands.annotations.permission.Permission;
 import java.time.Duration;
-import java.time.Instant;
 import java.util.UUID;
 import org.bukkit.entity.Player;
-import panda.utilities.text.Formatter;
 
+@Permission("eternalcombat.tagout")
```
Comment (P1otrulla) [inline eternalcombat-plugin/src/main/java/com/eternalcode/combat/fight/tagout/FightTagOutCommand.java:15]: unrelated 😙

**Review by P1otrulla — APPROVED** [review]
Dobra robota!

File: eternalcombat-plugin/src/main/java/com/eternalcode/combat/fight/tagout/FightTagOutCommand.java (hunk 2 above)
Comment (CitralFlo) [reply eternalcombat-plugin/src/main/java/com/eternalcode/combat/fight/tagout/FightTagOutCommand.java:15]: following simple rule: leave code better than was
